docs/UserGuide.md

```diff
@@ -170,81 +170,125 @@ Examples:
   * Updates the photo for the 2nd person with a new image from the specified path.
 
 
-###  Search contact
+###  Find contacts
 
-Generally, search commands will contain the following tokens:
+Lists contacts whose fields match the specified the specified find expression.
```

Comment:
perhaps use `FIND_EXPRESSION` instead? just a suggestion to follow what you entered under format.